runtime/Cargo.toml

```diff
@@ -11,66 +11,68 @@ version = '2.0.0'
 targets = ['x86_64-unknown-linux-gnu']
 
 [build-dependencies]
-substrate-wasm-builder = "4.0.0"
+substrate-wasm-builder = '4.0.0'
```

Comment:
Are the single vs double quotes something your editor does or from the template? I don't mind either but would be nice to avoid tools changing things back and forth making harder to review dependency changes.